src/Queue/Server.php

```diff
@@ -107,37 +195,35 @@ public function onWorkerStart(callable $callback): self
 
     /**
      * Is called when a Worker receives a Job.
-     * @param callable $callback
      * @return self
      */
-    public function onJob(callable $callback): self
+    public function onJob(): self
```

Comment:
```suggestion
    public function onJob(): self
```

We can keep everything consistent with Utopia style and ignore the `on` prefix

Reply:
will do